File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] newA = new int[nums.length * 2];

        for(int i = 0; i<nums.length; i++){
            newA[i]=nums[i];
            
           newA[i+nums.length] = nums[i];
           

        } return newA;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:

        counter = 0
        max_count = 0

        for n in nums:
            if n == 1:
                counter+=1
                max_count = max(max_count , counter)
            if n == 0:
               
                counter = 0
                
        return max_count






        